config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -3694,12 +3690,20 @@
 # DEFAULT: false
 # samigo.removeSubmission.restricted=true
 
+# SAK-46501 property to select the default sorting column in assessment list view
+# The column by which we want to sort the table
+# DEFAULT: 2
+# samigo.assessmentSortingColumn=9
+
 #########################################
-# SAMIGO SECURE DELIVERY (PROCTORING
+# SAMIGO SECURE DELIVERY (PROCTORING)
 #########################################
 
-# Respondus will send a closed-source JAR that will need to be installed in lib/ dir
-# samigo.secureDeliveryPlugins=/path/to/custom.jar
+# A list of JAR files or names for built-in modules separated by ":".
+# Each JAR file plugin can provide one or more secure module implementation(s).
+# Names for built-in modules are Proctorio and SafeExamBrowser
+# DEFAULT: SafeExamBrowser
```

Comment:
default should be none

Reply:
Hey Sam, default should be SafeExamBrowser in this branch, but should be none in the community version.